invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/subcommunity/browse.js

```diff
@@ -8,21 +8,33 @@
 
 import React from "react";
 import ReactDOM from "react-dom";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
 import _get from "lodash/get";
 
 import { CommunitiesCardGroup } from "@js/invenio_communities/community";
+import { Message } from "semantic-ui-react";
 
 const subCommunitiesContainer = document.getElementById("subcommunities-container");
 const domContainer = document.getElementById("invenio-browse-config");
 const apiEndpoint = _get(domContainer.dataset, "apiEndpoint");
 
 if (subCommunitiesContainer) {
-  ReactDOM.render(
-    <CommunitiesCardGroup
-      fetchDataUrl={apiEndpoint}
-      emptyMessage="There are no child communities."
-      defaultLogo="/static/images/square-placeholder.png"
-    />,
-    subCommunitiesContainer
-  );
+  if (apiEndpoint.length) {
```

Comment:
This change is not clear to me. Is the `apiEndpoint` unset when there are no sub communities?

Reply:
yep, we only include the 'subcommunities' endpoint to the links [when the community is allowed to have children](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/communities/services/config.py#L119) which means that the apiEndpoint is otherwise empty